llvm/test/Transforms/LoopStrengthReduce/lsr-term-fold.ll

```diff
@@ -1,5 +1,5 @@
 ; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
-; RUN: opt < %s -passes="loop-reduce" -S -lsr-term-fold | FileCheck %s
+; RUN: opt < %s -passes="loop-reduce,loop-term-fold" -S | FileCheck %s
```

Comment:
These tests probably shouldn't run loop-reduce first?

Reply:
Long term, yes, but for now this was the easiest way to be sure I wasn't introducing changes.

(To be clear, I plan on splitting these, I just want to do it in a separate commit.)